src/batch-client/types.ts

```diff
@@ -138,7 +140,10 @@ export interface SearchOptions {
 }
 
 export interface ShareInfosOptions {
```

Comment:
shareInfoOptions

Reply:
should I change the first letter to small 's'?

Reply:
my bad it should be ShareInfoOptions
change plural to singular form

Reply:
Done